packages/documentation/gatsby-config.js

```diff
@@ -66,5 +66,23 @@ module.exports = {
     `gatsby-transformer-remark`,
     `github-api-pages`,
     `mdx-pages`,
+    {
+      resolve: `@andrew-codes/gatsby-plugin-elasticlunr-search`,
```

Comment:
You might want to try https://github.com/gatsby-contrib/gatsby-plugin-elasticlunr-search. It looks like it's a fork that will work with Gatsby 2, which we're on.